test/ThemeProvider_test.jsx

```diff
@@ -9,7 +9,7 @@ describe('<ThemeProvider>', () => {
     const wrapper = shallow(
       <ThemeProvider name="tropical">
         <div>Under the sea</div>
-      </ThemeProvider>
+      </ThemeProvider>,
```

Comment:
You may have the right version locally installed, but package.json says 2.0.0 and we're on 2.1.1